lighthouse-cli/test/smokehouse/dobetterweb/run-tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+node lighthouse-cli/test/fixtures/static-server.js &
+
+sleep 0.5s
+
+config="lighthouse-core/config/dobetterweb.json"
+expectations="lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js"
```

Comment:
we should probably make a generic test runner that fires up the server and runs smokehouse. These long strings will still have to live somewhere though (though now I'm wondering if expectations files should have the config file path embedded in them)